resources/tentative/todomvc/src/components/todo-bottombar/todo-bottombar.component.js

```diff
@@ -0,0 +1,96 @@
+import template from "./todo-bottombar.template.js";
+
+class TodoBottombar extends HTMLElement {
+    static get observedAttributes() {
+        return ["total-items", "active-items"];
+    }
+
+    constructor() {
+        super();
+
+        // elements
+        this.element = undefined;
+        this.clearCompletedButton = undefined;
+        this.todoStatus = undefined;
+        this.filterLinks = undefined;
+        // state
+        this.connected = false;
+
+        this.clearCompletedItems = this.clearCompletedItems.bind(this);
+
+        this.shadow = this.attachShadow({ mode: "open" });
+    }
+
+    updateDisplay() {
+        if (this["total-items"] !== "0")
```

Comment:
It's somewhat odd to do string comparison like this.
Maybe `parseInt(this["total-items"])` or `+this["total-items"]` first?